carma/launch/environment.launch.py

```diff
@@ -305,7 +305,9 @@ def generate_launch_description():
                         ("external_objects", "external_object_predictions"),
                         ("external_objects_viz", "fused_external_objects_viz")
                     ],
-                    parameters=[ object_visualizer_param_file, vehicle_config_param_file ]
+                    parameters=[object_visualizer_param_file, vehicle_config_param_file,
+                                {'pedestrian_icon_path': "file:///opt/carma/vehicle/calibration/visualization_meshes/pedestrian.stl"}
```

Comment:
Can we change this to `{'pedestrian_icon_path': ['file:///', vehicle_calibration_dir, '/visualization_meshes/pedestrian.stl']}`?

It will require [1] passing `vehicle_calibration_dir` to environment.launch.py from carma_src.launch.py and [2] creating a `vehicle_calibration_dir` launch argument in this environment.launch.py file.